packages/web/lib/export/export-service.ts

```
@@ -0,0 +1,184 @@
+
+    if (!options.format) {
+      errors.push('Export format is required')
+    } else if (!this.getSupportedFormats().includes(options.format)) {
+      errors.push(`Unsupported format: ${options.format}`)
+    }
+
+    // Format-specific validation
+    switch (options.format) {
+      case 'pdf':
+        const pdfOptions = options as any
+        if (
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)